assets/js/components/Dashboard/Dashboard.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { ComponentType, useEffect } from "react";
+import React, { ComponentType, useEffect, useState } from "react";
```

Comment:
So that logic acts as `firstOrDefault` function (which unfortunately TS does not have). The array of alerts won't ever include `undefined` values but it could be empty. If it is empty, we can't just reference the first item in the array because there isn't one. If we do `.find((alert) => alert !== undefined)`, we can safely get the first item in the array regardless of the `length`. So `[].find((alert) => alert !== undefined)` would return `undefined`. `["123"].find((alert) => alert !== undefined)` would return `123`.

Reply:
Ah, I see. I think that can be condensed to `.find((alert) => alert)`?